LICENSE.md

```diff
@@ -1,22 +1,21 @@
-The Documenter.jl package is licensed under the MIT "Expat" License:
+MIT License
 
-> Copyright (c) 2016: Michael Hatherly.
-> 
-> Permission is hereby granted, free of charge, to any person obtaining a copy
-> of this software and associated documentation files (the "Software"), to deal
-> in the Software without restriction, including without limitation the rights
-> to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-> copies of the Software, and to permit persons to whom the Software is
-> furnished to do so, subject to the following conditions:
-> 
-> The above copyright notice and this permission notice shall be included in all
-> copies or substantial portions of the Software.
-> 
-> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-> IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-> FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-> AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-> LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-> OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-> SOFTWARE.
-> 
+Copyright (c) 2016-2021: Michael Hatherly, Morten Piibeleht and Fredrik Ekre.
```

Comment:
Maybe
```suggestion
Copyright (c) 2016-2021: Michael Hatherly, Morten Piibeleht, Fredrik Ekre, and Documenter.jl contributors
```
?
Or just 
```suggestion
Copyright (c) 2016-2021: Documenter.jl contributors
```
?
I don't really know how copyright works for a collaborative effort like this though. Does the name "matter"? Are everyone that contributed part of the copyright?

Reply:
I'm not sure, but the [Julia repository](https://github.com/JuliaLang/julia/blob/master/LICENSE.md) is similar to the first one. So, I support the first one.